src/iap/shared/components/carousel/carousel.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="carousel">
```

Comment:
:point_right: :point_left: 
```suggestion
<div class="iap-carousel">
```